src/scripts/Common/Base.cpp

```diff
@@ -679,6 +679,53 @@ EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, const char* pTex
     return NewEmote;
 }
 
+EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, uint32 scripttext)
+{
+    EmoteDesc* NewEmote = NULL;
+    NpcScriptText const* ct = sMySQLStore.GetNpcScriptText(scripttext);
+    TextType pType;
+
+    switch (ct->type)
+    {
+    case CHAT_MSG_SAY:
```

Comment:
A case label should be indented from its switch statement.
http://www.ascemu.org/wiki/index.php?title=Coding_standards#indent